src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+import mate.academy.exceptions.DataProcessingException;
+
+public class ConnectionUtil {
+    private static final Properties DB_PROPERTIES;
+    private static final String URL = "jdbc:mysql://localhost:3306/mate";
```

Comment:
The URL seems to be incomplete. It should specify the database that you are connecting to. For example, 'jdbc:mysql://localhost:3306/your_database_name'.